packages/nextjs/src/performance/client.ts

```diff
@@ -0,0 +1,115 @@
+import { Primitive, Transaction, TransactionContext } from '@sentry/types';
+import { fill, getGlobalObject } from '@sentry/utils';
+import { default as Router } from 'next/router';
+
+const global = getGlobalObject<Window>();
+
+type StartTransactionCb = (context: TransactionContext) => Transaction | undefined;
+
+const DEFAULT_TAGS = Object.freeze({
+  'routing.instrumentation': 'next-router',
+});
+
+const QUERY_PARAM_REGEX = /\?(.*)/;
+
+let activeTransaction: Transaction | undefined = undefined;
+let prevTransactionId: string | undefined = undefined;
```

Comment:
let's rename this to `prevTransactionName` to make it more accurate.

Reply:
Not related to this PR, but having a `name` that's a "[Human-readable] identifier" doesn't make things very consistent.